Foundation/NSGeometry.swift

```diff
@@ -25,6 +25,20 @@ public struct CGPoint {
     }
 }
 
+extension CGPoint {
```

Comment:
Ah I was just mimicing the way it's structured in `Foundation` on **macOS**.
It also made sense to me to keep this syntax-sugar-like initializers in an `extension`, just for the sake of visual cue.

These are the reasons I did it.